postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -527,14 +546,7 @@ def _run_explain(self, dbname, statement, obfuscated_statement):
                 if not result or len(result) < 1 or len(result[0]) < 1:
                     return None
                 return result[0][0]
-            except Exception as e:
-                self._check.count(
-                    "dd.postgres.run_explain.error",
-                    1,
-                    tags=self._dbtags(dbname, "error:explain-database_error-{}".format(type(e)))
-                    + self._check._get_debug_tags(),
-                    hostname=self._check.resolved_hostname,
-                )
+            except Exception:
                 raise
```

Comment:
Is there a reasoning behind catching this exception if it's going to be immediately raised again?

Reply:
Ah, you're right. We were doing this before because we wanted to emit a metric, but that logic has moved: https://github.com/DataDog/integrations-core/pull/13224#discussion_r1007568056 -- Updating